basics/installation/install-with-docker.md

```diff
@@ -0,0 +1,471 @@
+
+- [Docker's website](https://www.docker.com/)
+- [PrestaShop's Docker Github repository](https://github.com/PrestaShop/docker)
+- [PrestaShop's Docker Hub](https://hub.docker.com/r/prestashop/prestashop/)
+
+The easiest way to begin with PrestaShop on Docker is to run your stack with Docker compose. It will ease container management, networks, volume persistence, ...
+
+## Install Docker
+
+Docker is available on each major operating system. [You will need to install Docker](https://docs.docker.com/get-docker/).
+
+{{% notice note %}}
+When using Windows (10+), it can be a good solution to [install WSL (Windows Subsystem for Linux)](https://learn.microsoft.com/en-gb/windows/wsl/install), and then install Docker on the WSL distribution. You will benefit of all the advantages of a Linux OS when using Shell / CLI.
+Note: WSL2 should be a better option when running Docker.
+{{% /notice %}}
+
+### Install docker compose
+
+Docker Compose is a tool that simplifies the management of multi-container Docker applications by allowing developers to define and configure complex application stacks using a declarative YAML file, enabling easy orchestration, scaling, and networking of containers with a single command.
+
+docker compose is included in versions of Docker Desktop (Windows, Mac). If you are using Docker on Linux, [you need to install docker compose](https://docs.docker.com/compose/install/). 
+
+
+## Use docker compose to manage your stack
+
+docker compose will help starting and stopping your PrestaShop stack, persist your database with a volume, persist your installation, and will allow you to mount local modules and/or themes in PrestaShop. 
+
+### Create the docker compose.yml manifest
+
+```yaml
+version: '3'
+services:
+  mysql:
+    container_name: some-mysql
+    image: mysql:5.7
+    restart: unless-stopped
+    environment:
+      MYSQL_ROOT_PASSWORD: admin
+      MYSQL_DATABASE: prestashop
+    networks:
+      - prestashop_network
+  prestashop:
+    container_name: prestashop
+    image: prestashop/prestashop:latest
+    restart: unless-stopped
+    depends_on:
+      - mysql
+    ports:
+      - 8080:80
+    environment:
+      DB_SERVER: some-mysql
+      DB_NAME: prestashop
+      DB_USER: root
+      DB_PASSWD: admin
+    networks:
+      - prestashop_network
+networks:
+    prestashop_network:
+```
+
+{{% notice info %}}
+When using a M1-chip Mac, you may add: `platform: linux/x86_64` on each container declaration. 
+{{% /notice %}}
+
+### Automatically install PrestaShop with test data
+
+{{% notice note %}}
+This is the easiest and quickest way to start and test PrestaShop with Docker.
+{{% /notice %}}
+
+You can automatically install PrestaShop, and add test data, without having to use the installation assistant when creating your Docker compose stack. 
+
+To achieve this, you need to set those environment variables in your PrestaShop container declaration in your  `docker-compose.yml` file:
+
+```yaml
+PS_DEMO_MODE: 1
```

Comment:
Just a question, are you sure it is necessary for the example data to install? Because there's something like the demo mode in PrestaShop software which is different.

Reply:
mmm let me check 👍